pkg/genericapiserver/genericapiserver.go

```diff
@@ -627,6 +630,11 @@ func (s *GenericAPIServer) Run(options *ServerRunOptions) {
 	glog.Fatal(http.ListenAndServe())
 }
 
+func fileExists(path string) bool {
+	_, err := os.Stat(path)
+	return !os.IsNotExist(err)
```

Comment:
This may be overly specific, e.g., what if the file is present but we don't have permission?